registry/storage/shares_test.go

```diff
@@ -200,6 +207,55 @@ func generateRandomValidatorShare(splitKeys map[uint64]*bls.SecretKey) (*ssvtype
 
 	quorum, partialQuorum := ssvtypes.ComputeQuorumAndPartialQuorum(len(splitKeys))
 
+	return &storageShare{
+		Share: Share{
+			OperatorID:          1,
+			ValidatorPubKey:     sk1.GetPublicKey().Serialize(),
+			SharePubKey:         sk2.GetPublicKey().Serialize(),
+			Committee:           ibftCommittee,
+			Quorum:              quorum,
+			PartialQuorum:       partialQuorum,
+			DomainType:          networkconfig.TestNetwork.Domain,
+			FeeRecipientAddress: common.HexToAddress("0xFeedB14D8b2C76FdF808C29818b06b830E8C2c0e"),
+			Graffiti:            bytes.Repeat([]byte{0x01}, 32),
+		},
+		Metadata: ssvtypes.Metadata{
+			BeaconMetadata: &beaconprotocol.ValidatorMetadata{
+				Balance:         1,
+				Status:          2,
+				Index:           3,
+				ActivationEpoch: 4,
+			},
+			OwnerAddress: common.HexToAddress("0xFeedB14D8b2C76FdF808C29818b06b830E8C2c0e"),
+			Liquidated:   true,
+		},
+	}, &sk1
+}
+
+func generateRandomValidatorSpecShare(splitKeys map[uint64]*bls.SecretKey) (*ssvtypes.SSVShare, *bls.SecretKey) {
+	threshold.Init()
+
+	sk1 := bls.SecretKey{}
+	sk1.SetByCSPRNG()
+
+	sk2 := bls.SecretKey{}
+	sk2.SetByCSPRNG()
+
+	var ibftCommittee []*spectypes.Operator
+	for operatorID, sk := range splitKeys {
+		pk, _, _ := rsaencryption.GenerateKeys()
```

Comment:
Although it's unlikely ever happens, I'd check the error, we can panic in this case